stack.yaml

```diff
@@ -10,4 +11,7 @@ packages:
   - ./persistent-qq
 
 extra-deps:
-    - lift-type-0.1.0.0
+  - aeson-2.0.2.0
+  # https://github.com/yesodweb/shakespeare/pull/260
```

Comment:
This is extraneous, from a bad cherry-pick. Resolved here: https://github.com/yesodweb/persistent/pull/1338/files#diff-0de01f189d384bfbe69e806ef70b1a425b27394dbe2a9d0e62ffd010dda86c03L15